main_logic/core.py

```
@@ -1311,9 +1589,9 @@ async def _process_stream_data_internal(self, message: dict):
         except web_exceptions.ConnectionClosedError as e:
             logger.error(f"💥 Stream: Error sending data to session: {e}")
```

Ruff (0.14.10)

Use `logging.exception` instead of `logging.error`

Replace with `exception`

(TRY400)